lib/calendars/england_and_wales.yml

```diff
@@ -0,0 +1,16 @@
+holidays:
+  # Taken from: https://www.gov.uk/bank-holidays
+
+  # 2022
+  - 2022-12-26
+  - 2022-12-27
+
+  # 2023
+  - 2023-01-02
+  - 2023-04-07
+  - 2023-04-10
+  - 2023-05-01
+  - 2023-05-29
+  - 2023-08-28
+  - 2023-12-25
+  - 2023-12-26
```

Comment:
We should add something about this to the docs. We need to think about how we'll remember to add 2024 🤔

Reply:
Yes, agreed, I wasn't really sure where to capture updating this... a card on the backlog?